source/tools/watcher.py

```diff
@@ -19,10 +19,13 @@
 Watcher module for config cluster
 """
 
+import os
 import sys
 import time
 import uuid
 import logging
+from ConfigParser import RawConfigParser
+from StringIO import StringIO
 from source.tools.log_handler import LogHandler
```

Comment:
First the ```import foo```, ordered by: length, alphabet
Then the ```from foo import bar``` from the non-ovs libraries, ordered by alphabet
Third the ```from foo import bar``` from the ovs libraries, ordered by alphabet

So that one is good